liquidctl/driver/aura_led.py

```diff
@@ -0,0 +1,322 @@
+    "end_effect": [_CMD_CODE, 0x35, 0x00, 0x00, 0x00, 0xFF],
+    "start_seq1": [_CMD_CODE, 0x35],
+    "start_seq2": [_CMD_CODE, 0x36, 0x00],
+    "end_seq1": [_CMD_CODE, 0x35, 0x00, 0x00, 0x01, 0x05],
+    "end_seq2": [_CMD_CODE, 0x3F, 0x55, 0x00, 0x00],
+    "reset_seq1": [_CMD_CODE, 0x38, 0x01, 0x01],
+    "reset_seq2": [_CMD_CODE, 0x38, 0x00, 0x01],
+    "channel_off_pre1": [_CMD_CODE, 0x38, 0x01, 0x00],
+    "channel_off_pre2": [_CMD_CODE, 0x38, 0x00, 0x00],
+    "channel_off_prefix": [_CMD_CODE, 0x35, 0x01, 0x00, 0x00, 0xFF],
+    "firmware": [_CMD_CODE, 0x82],
+    "config": [_CMD_CODE, 0xB0],
+}
+
+# channel_type 0 designates RGB bus
+# channel_type 1 designates ARGB bus
+# "effect" mode channel IDs are different from "direct" mode channel IDs
+_ColorChannel = namedtuple(
+    "_ColorChannel", ["name", "channel_id", "direct_channel_id", "channel_type", "rgb_offset"]
+)
+
+_COLOR_CHANNELS = {
+    channel.name: channel
+    for channel in [
+        _ColorChannel("rgb", 0x01, 0x00, 0x00, 0x0),
+        _ColorChannel("argb1", 0x02, 0x00, 0x01, 0x01),
+        _ColorChannel("argb2", 0x04, 0x01, 0x01, 0x02),
+        _ColorChannel("argb3", 0x08, 0x02, 0x01, 0x03),
+    ]
+}
+
+_ColorMode = namedtuple("_ColorMode", ["name", "value", "takes_color"])
+
+_COLOR_MODES = {
+    mode.name: mode
+    for mode in [
+        _ColorMode("off", 0x00, takes_color=False),
+        _ColorMode("static", 0x01, takes_color=True),
+        _ColorMode("breathing", 0x02, takes_color=True),
+        _ColorMode("flashing", 0x03, takes_color=True),
+        _ColorMode("spectrum_cycle", 0x04, takes_color=False),
+        _ColorMode("rainbow", 0x05, takes_color=False),
+        _ColorMode("spectrum_cycle_breathing", 0x06, takes_color=False),
+        _ColorMode("chase_fade", 0x07, takes_color=True),
+        _ColorMode("spectrum_cycle_chase_fade", 0x08, takes_color=False),
+        _ColorMode("chase", 0x09, takes_color=True),
+        _ColorMode("spectrum_cycle_chase", 0x0A, takes_color=False),
+        _ColorMode("spectrum_cycle_wave", 0x0B, takes_color=False),
+        _ColorMode("chase_rainbow_pulse", 0x0C, takes_color=False),
+        _ColorMode("rainbow_flicker", 0x0D, takes_color=False),
+        _ColorMode("gentle_transition", 0x10, takes_color=False),
+        _ColorMode("wave_propagation", 0x11, takes_color=False),
+        _ColorMode("wave_propagation_pause", 0x12, takes_color=False),
+        _ColorMode("red_pulse", 0x13, takes_color=False),
+    ]
+}
+
+
+class AuraLed(UsbHidDriver):
+    """
+    liquidctl driver for Asus Aura LED USB controllers.
+    This driver only supports 'effect' mode, hence no speed/color channels
+
+    Devices 0x1939 and 0x18F3 are not fully supported at this time; users are welcome
+    to uncomment the 2nd and/or 3rd line below in SUPPORTED_DEVICES if they wish to
+    experiment with this driver and provide feedback
+    """
+
+    SUPPORTED_DEVICES = [
+        (0x0B05, 0x19AF, None, "AsusTek Aura LED Controller", {}),
```

Comment:
This device should have `(experimental)` in the name, see https://github.com/liquidctl/liquidctl/blob/71aca8703c0326a492f41191d504006c6aaf1564/liquidctl/driver/commander_core.py#L54 for an example.

Reply:
Done.

Reply:
The `status` command now returns this in the absence of the `--debug` flag:

```
% liquidctl -m Aura status
AsusTek Aura LED Controller (experimental)
├── ARGB channels: 2      
└──  RGB channels: 1
```